tools/FakeItEasy.Build/Program.cs

```diff
@@ -110,10 +111,23 @@ public static void Main(string[] args)
             RunTargets(args);
         }
 
-        private static void RunTests(string testDirectory)
+        private static IEnumerable<(string directory, string framework)> GetDirectoriesAndFrameworks(string[] directories) =>
+            from directory in directories from framework in GetFrameworks(directory) select (directory, framework);

```

Comment:
I know the tuples are convenient, but I don't love the `Bullseye/unit/tests/(directory, framework)` output. It's disjointed.
With a tiny custom class and overridden `ToString`, we could have

![image](https://user-images.githubusercontent.com/3275797/45328068-26dae880-b528-11e8-8cb9-0db5e5f86bcb.png)

Reply:
Yeah, it would be even better